cmake/OpenCVDetectCUDA.cmake

```diff
@@ -123,7 +125,7 @@ if(CUDA_FOUND)
       if(${CUDA_VERSION} VERSION_LESS "9.0")
         set(__cuda_arch_bin "2.0 3.0 3.5 3.7 5.0 5.2 6.0 6.1")
       else()
-        set(__cuda_arch_bin "3.0 3.5 3.7 5.0 5.2 6.0 6.1 7.0")
+        set(__cuda_arch_bin "3.0 3.5 3.7 5.0 5.2 6.0 6.1 7.0 7.5")
```

Comment:
Does CUDA 9.x supports compute capability 7.5?

Reply:
No, let me push another elseif to guard this real quick